### PR TITLE
Add website guidelines

### PR DESCRIPTION
**Pull request by liggitt**
Added a brief policy on use/duration of the website banner for community statements as discussed in steering. I wasn't quite sure where this belonged, but under "communication", peer to mailing list / slack / discuss / social media guidelines seemed reasonable. All suggestions on location and phrasing welcome.

Also linked to existing sig-docs website content/style guides, and the blog subproject / contribution page.

/cc @cblecker @derekwaynecarr @dims @mrbobbytables @nikhita @parispittman

steering:
- [ ] @cblecker
- [x] @derekwaynecarr
- [x] @dims
- [x] @liggitt
- [x] @mrbobbytables
- [x] @nikhita
- [ ] @parispittman

sig-docs chairs:
- [x] @irvifa
- [x] @jimangel
- [ ] @kbarnard10

sig-docs tech leads:
- [x] @kbhawkey
- [x] @onlydole
- [x] @sftim

### CODE REVIEW

**Comment by nikhita** [comment]
cc @irvifa @jimangel @kbarnard10 @kbhawkey @onlydole @sftim @zacharysarah

**Comment by sftim** [comment]
LGTM

**Review by irvifa — COMMENTED** [review]
/lgtm

**Comment by dims** [comment]
LGTM (with steering hat on)

File: communication/website-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Website guidelines
+
+## General content
+
+The sig-docs [website subproject] maintains the kubernetes.io website,
```
Comment (kbhawkey) [inline communication/website-guidelines.md:5]: nit ?: s/The sig-docs/SIG Docs/

Comment (liggitt) [reply communication/website-guidelines.md]: updated to SIG-Docs to match usage under https://github.com/kubernetes/community/tree/master/sig-docs

**Comment by liggitt** [comment]
replaced `sig-docs` with `SIG-Docs` - https://github.com/kubernetes/community/compare/25423c12f23216dccdbcf45a87682136a11c20ab..5148c9a6de7af60d7a1eb64c4040f49be0c4a607

File: communication/website-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The [Kubernetes Blog] is a subproject of SIG-Docs and operated by the [blog team].
+
+[Submit a blog post]
+
+## Community statements
+
+Prominent notices on Kubernetes web sites (such as site-wide or front-page banners)
+may be used occasionally to publish statements on behalf of the Kubernetes community.
+The content of those statements must be approved by the [Kubernetes Steering Committee].
```
Comment (kbhawkey) [inline communication/website-guidelines.md:22]: I'd like to move away from using the word "statements" and instead choose another word such as announcements.
Or expand your description to include statements and announcements if you think they are different things.

nit?: s/statements/announcements/
nit, line 22: Could simplify, The content must be approved ... by opening a pull request with the proposed content? 
nit, line 23: Content in the form of a site wide banner may remain ...

Comment (liggitt) [reply communication/website-guidelines.md:22]: I do think statements and announcements are distinct. If there's agreement we want to treat them the same way and include statements and announcements using the banner in this policy, I can update. Thoughts from other reviewers?

Comment (sftim) [reply communication/website-guidelines.md:22]: I was thinking about how we might implement this - maybe two different kinds of Markdown file and a `kind: statement` or `kind: announcement` in the front matter (not the meaning of `kind` as in the Kubernetes API, but broadly the same idea).
My guess is that announcements are _always_ time limited and are always linked to an upcoming or recent event, whereas statements might not be.

As a website reviewer, I hope it'll be obvious which changes need steering committee approval and which don't.

Comment (mrbobbytables) [reply communication/website-guidelines.md:22]: I would lean towards treating them the same way. In an event where an announcement and a previously published statement may clash it should still be bubbled up as to which should take precedence in addition to steering visibility of such a change.

Comment (nikhita) [reply communication/website-guidelines.md:22]: Maybe we can just use `notice` instead of separating them into announcements or statements?

Comment (derekwaynecarr) [reply communication/website-guidelines.md:22]: +1 to @nikhita usage of term `notice`.  Per our steering discussion, I thought we intentionally did not want to treat a statement and announcement as separate concepts, and more just have a consistent practice for the banner as a whole.

Comment (liggitt) [reply communication/website-guidelines.md:22]: notice sounds good to me, as does removing ambiguity about what category a given banner falls into

Comment (liggitt) [reply communication/website-guidelines.md]: Updated with "notice". I don't think the mechanics need to be spelled out here, but I'd imagine banner content living in a dir in the website repo with the right groups as approvers

**Comment by jimangel** [comment]
/lgtm

Thanks!

File: communication/website-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Blog 
+
+The [Kubernetes Blog] is a subproject of SIG-Docs and operated by the [blog team].
+
+[Submit a blog post]
+
+## Community statements
+
+Prominent notices on Kubernetes web sites (such as site-wide or front-page banners)
```
Comment (kbhawkey) [inline communication/website-guidelines.md:20]: nit: I assume that these guidelines apply to _any_ Kubernetes community-related website (not only `kubernetes.io`).
nit: s/web site/website/
/lgtm

**Comment by nikhita** [comment]
LGTM from me on the content. I think @parispittman had opinions on where we should house this doc.

/assign @parispittman

**Pull request merged**